src/openshift/cluster.ts

```diff
@@ -143,6 +146,23 @@ export class Cluster extends OpenShiftItem {
             .catch((error) => Promise.reject(`Failed to login to cluster '${clusterURL}' with '${error}'!`));
     }
 
+    static async readFromClipboard() {
+        return await clipboardy.readSync();
+    }
+
+    static async ocLogin(skipConfirmation: boolean = false): Promise<string> {
+        const response = await Cluster.requestLoginConfirmation(skipConfirmation);
+        if (response !== 'Yes') return null;
+        const oclogin = Cluster.ocLoginCommandMatches(await clipboardy.readSync());
+        if (!oclogin) throw Error('oc login command not found in clipboard');
+        const clusterURL = Cluster.clusterURL(oclogin);
+        if (!clusterURL) return null;
+        return Promise.resolve()
```

Comment:
I don't understand how it works cause you're not extracting the token but only the cluster URL, something is missing here

Reply:
OK I got it as you're running the whole oc command line. See with @dgolovin as I think we try to lower our dependency on `oc` binary